app/controllers/users_controller.rb

```diff
@@ -18,10 +21,33 @@ def create
     end
   end
 
+  def edit
+  end
+
+  def update
+    if @user.update_attributes(user_params)
+      flash[:success] = "Profile updated"
```

Comment:
[rubocop] Prefer single-quoted strings when you don't need string interpolation or special symbols.